source/slang/slang-ir-dce.h

```diff
@@ -12,6 +12,7 @@ namespace Slang
         bool keepExportsAlive = false;
         bool keepLayoutsAlive = false;
         bool useFastAnalysis = false;
+        bool keepGlobalParamsAlive = true;
```

Comment:
Shouldn't this be false by default. It seems little inconsistent to the lines above.

Reply:
true is more conservative than false, and it is safer not to remove something by default.